server/validators/user_validator.go

```diff
@@ -25,8 +25,7 @@ func ValidateMail(v interface{}, param string) error {
 	if st.Kind() != reflect.String {
 		return errors.New("ValidateMail only validates strings")
 	}
-	_, err := mail.ParseAddress(st.String())
-	return err
+	return ValidMail(st.String())
```

Comment:
Let's make all of the errors lowercased in `ValidateMail`, and `ValidateSSHKey`, .. etc